File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Precancel/ConfirmationID.vb
Imports Newtonsoft.Json

Namespace ExpediaRapid.SerializableClasses.BookingItinerary
    Public Class ConfirmationID

        <JsonProperty("expedia")>
        Public Property ExpediaConfirmationID As String

        <JsonProperty("property")>
        Public Property PropertyConfirmationID As String

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Precancel/ConfirmationID.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses.BookingItinerary
{
    using Newtonsoft.Json;

    public class ConfirmationID
    {

        [JsonProperty("expedia")]
        public string ExpediaConfirmationID { get; set; }

        [JsonProperty("property")]
        public string PropertyConfirmationID { get; set; }

    }

}